README.md

```diff
@@ -34,6 +45,38 @@ This version of the extension provides Bootstrap 4.3.1.
 **Remark:** It is _NOT_ necessary to install or load any extensions this extensions
 depends on.
 
+### Installation without Composer
```

Comment:
I'm confus. This section is full of Composer stuff, and so is the above one.

Reply:
Installation instructions for Composer do not need to include explanation of composer.local.json btw: https://github.com/ProfessionalWiki/chameleon/blob/master/docs/installation.md#installation-with-composer

Reply:
Agreed, in both cases the SCSS library is installed using composer, so it cannot really be said that the "without composer" instructions are truly without composer. I could change the titles of the two sections ("Installation of Bootstrap with Composer" and "Installation of Only SCSS with Composer"??)? I'm struggling with making the instructions concise and useful without getting sucked into the "only libraries should be installed with composer" argument.

The reference to composer.local.json in the "Installation with Composer" section were in the original instructions. I left them there, because it is useful to distinguish between what you would put in that file if you are installing all of Bootstrap with composer versus if you are only installing SCSS with composer. I tested the instructions for both cases and made sure that there were no steps missing. The Chameleon instructions you point to should probably be amended similarly: if one were to run `composer update` in the MediaWiki directory (in the process of updating another library, perhaps) without adding Chameleon to composer.local.json, Chameleon would be uninstalled.

Reply:
> The Chameleon instructions you point to should probably be amended similarly: if one were to run composer update in the MediaWiki directory (in the process of updating another library, perhaps) without adding Chameleon to composer.local.json, Chameleon would be uninstalled.

The first command in the Chameleon instructions adds Chameleon to composer.local.json. You end up with the same stuff as if you where to manually edit that file.

Reply:
Oh, cool! I completely failed to notice the 'COMPOSER=composer.local.json' bit. That is indeed a better solution. I'll fix that here.

Reply:
Oh, cool! I completely failed to notice the 'COMPOSER=composer.local.json' bit. That is indeed a better solution. I'll fix that here.